redwood-treehouse-host/build.gradle

```diff
@@ -0,0 +1,57 @@
+import app.cash.redwood.buildsupport.KmpTargets
+
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
+apply plugin: 'org.jetbrains.kotlin.plugin.serialization'
+apply plugin: 'com.android.library'
+apply plugin: 'com.vanniktech.maven.publish'
+apply plugin: 'org.jetbrains.dokka' // Must be applied here for publish plugin.
+apply plugin: 'app.cash.zipline'
+
+kotlin {
+  KmpTargets.addAllTargets(project, true /* skipJs */)
```

Comment:
We can’t host Treehouse on JS because we don’t have a JS implementation of ZiplineLoader.